adam-core/src/main/scala/org/bdgenomics/adam/util/FileExtensions.scala

```diff
@@ -63,7 +63,11 @@ private[adam] object FileExtensions {
    */
   def isFastqExt(pathName: String): Boolean = {
     pathName.endsWith(".fq") ||
-      pathName.endsWith(".fastq")
+      pathName.endsWith(".fastq") ||
+      pathName.endsWith(".fastq.gz") ||
```

Comment:
Actually, didn't even need a modification to `ADAMContext.trimExtensionIfCompressed`.